app/javascript/components/forms/admin/EditRoleForm.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Stack } from 'react-bootstrap';
+import { useForm } from 'react-hook-form';
+import { editRoleFormConfig, editRoleFormFields } from '../../../helpers/forms/EditRoleFormHelpers';
+import Form from '../Form';
+import FormControl from '../FormControl';
+import Spinner from '../../shared/stylings/Spinner';
+
+export default function EditRoleForm({ role }) {
+  const { defaultValues } = editRoleFormConfig;
+  defaultValues.name = role.name;
+
+  const methods = useForm(editRoleFormConfig);
+  const { isSubmitting } = methods.formState;
```

Comment:
Shouldn'y we be using the mutation here?

Reply:
Yup, forgot to change it.